wire-library/wire-runtime/src/commonMain/kotlin/com/squareup/wire/Empty.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire
+
+/**
+ * A generic empty message that you can re-use to avoid defining duplicated empty messages in your
+ * APIs.
+ */
+object Empty
```

Comment:
If this doesn't have to be a subtype of `Message` can we use `kotlin.Unit`?

Reply:
We can!

Reply:
fixed